### PR TITLE
Keyboard focus

### PR DESCRIPTION
**Pull request by RobertoCarrilloAvila**
## Screenshots:
https://github.com/RobertoCarrilloAvila/target/assets/16627969/f02b6412-470d-4024-bdea-03116d5b267b

### CODE REVIEW

File: src/global.scss [hunk 1]
```diff
@@ -29,3 +29,7 @@
     stroke-width: 1px;
   }
 }
+
+button:focus-visible, a:focus-visible {
+  outline: -webkit-focus-ring-color auto 1px !important;
```
Comment (afmicc) [inline src/global.scss:34]: what is the default appearance vs the new one? shouldn't the default be more accessible than a customization given it's what the users get used to seeing?

Comment (RobertoCarrilloAvila) [reply src/global.scss:34]: the `a` tag was working however for the `button` tag it didn't, So I included in order to identify when the focus is on the buttons

**Review by bmarcio20 — APPROVED** [review]
I see that this solution works, however, I think it might not scale so good (imagine the web now has more than 3 elements in the menu, multiple refs will be needed and logic will vary on each case)
I would propose to try to find a more "native" way of browsing with the keyboard for the future.

**Review by Ivan-Encinas — COMMENTED** [review]
Maybe you can take a look to React-Aria to handle this keyboard actions and other accessibility behaviors

**Comment by RobertoCarrilloAvila** [comment]
@bmarcio20 @Ivan-Encinas Can you check again? I changed a bit the way I was implemented.
Now i used css to change visibility and it works as well

**Pull request merged**